packages/tools/src/utilities/orientation/getOrientationString.ts

```diff
@@ -0,0 +1,52 @@
+import { Types } from '@cornerstonejs/core';
+
+/**
+ * Returns the orientation of the vector in the patient coordinate system.
+ * @public
+ *
+ * @param vector - Input array
+ * @returns The orientation in the patient coordinate system.
+ */
+export default function getOrientationString(vector: Types.Point3): string {
+  // Thanks to David Clunie
+  // https://sites.google.com/site/dicomnotes/
+
+  let orientation = '';
+  const orientationX = vector[0] < 0 ? 'R' : 'L';
```

Comment:
I understand this is just pulled from CSTLegacy, but we have to be really careful here!

This method is actually get DICOM Orientation string, it assumes we are in LPS. This is not generic and will show the incorrect values if the volume is RAS, for example. We need to be very explicit what this helper is for.

Also its weird to me that have F and H (foot and head?) for Inferior (I) and Superior (S). Was there a good reason for this originally @swederik ?